src/test/java/guitests/DialogMessageTests.java

```diff
@@ -40,6 +40,25 @@ public void showYesNoWarningDialogTest() throws ExecutionException, InterruptedE
     }
 
     @Test
+    public void showYesNoConfirmationDialog_tryYesAndNo_getCorrectValue() throws ExecutionException, InterruptedException {
+        FutureTask<Boolean> yesTask = new FutureTask<>(() ->
+                DialogMessage.showYesNoConfirmationDialog("Confirm", "Confirm Header", "Confirm Message", "yEs", "nO"));
+        Platform.runLater(yesTask);
+        waitUntilNodeAppears(hasText("Confirm Header"));
+        assertNodeExists(hasText("Confirm Header"));
+        clickOn("yEs");
```

Comment:
I was thinking usin something more readable, say 1 test is `yes/no`,
another is `Non-Standard-Yes/Non-Standard-No`. But a comment works, too.
On Apr 9, 2016 10:02 AM, "Damith C. Rajapakse" <notifications@github.com>
wrote:

> In src/test/java/guitests/DialogMessageTests.java
> <https://github.com/HubTurbo/HubTurbo/pull/1485#discussion_r59108223>:
>
> > @@ -40,6 +40,25 @@ public void showYesNoWarningDialogTest() throws ExecutionException, InterruptedE
> >      }
> >
> >      @Test
> > +    public void showYesNoConfirmationDialog_tryYesAndNo_getCorrectValue() throws ExecutionException, InterruptedException {
> > +        FutureTask<Boolean> yesTask = new FutureTask<>(() ->
> > +                DialogMessage.showYesNoConfirmationDialog("Confirm", "Confirm Header", "Confirm Message", "yEs", "nO"));
> > +        Platform.runLater(yesTask);
> > +        waitUntilNodeAppears(hasText("Confirm Header"));
> > +        assertNodeExists(hasText("Confirm Header"));
> > +        clickOn("yEs");
>
> If it serves a purpose, we should at least make it clear (add a comment?)
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/HubTurbo/HubTurbo/pull/1485/files/7b0a8d2d74a153882115dd07ad60d30bf8d35e43#r59108223>
>